features/list_companies_of_category.feature

```diff
@@ -0,0 +1,36 @@
+Feature: As a visitor
```

Comment:
Yes -- you're seeing it. :-)  And there's a difference between describing a feature and doing the testing.  

The crux of it is: the feature needs to say "everyone" -- and so you also need to test for everyone.  (If the feature meant only visitors, then you have to test to ensure that only visitors can do it, and that users, members, and admins cannot.)